static/src/javascripts/projects/commercial/modules/prebid/bid-config.js

```diff
@@ -149,6 +151,39 @@ const getImprovePlacementId = (sizes: PrebidSize[]): number => {
     }
 };
 
+const getAppNexusPlacementId = (sizes: PrebidSize[]): string => {
+    switch (config.page.edition) {
+        case 'UK':
+            switch (getBreakpointKey()) {
+                case 'D':
+                    if (containsMpuOrDmpu(sizes)) {
+                        return '13366606';
+                    }
+                    if (containsLeaderboardOrBillboard(sizes)) {
+                        return '13366615';
+                    }
+                    return '';
+                case 'M':
+                    if (contains(sizes, [300, 250])) {
```

Comment:
Perhaps refactor `containsMpuOrDmpu` into `containsMpu or containsDmpu` and use `containsMpu` here?